src/test/scala/scalismo/faces/parameters/SphericalHarmonicsLightTests.scala

```diff
@@ -52,6 +53,18 @@ class SphericalHarmonicsLightTests extends FacesTestSuite {
       band0.coefficients(0) shouldBe sh.coefficients(0)
     }
 
+    it("SH to DenseVector and from DenseVector"){
+      val sh = SphericalHarmonicsLight.frontal
+      val shB = sh.toBreezeVector
+      val shN = SphericalHarmonicsLight.fromBreezeVector(shB)
+      sh.toBreezeVector.toArray should contain theSameElementsInOrderAs  shN.toBreezeVector.toArray
```

Comment:
I would argue to test here directly `sh shouldBe shN`. The additional conversion might hide some problems.